ironfish-cli/src/commands/wallet/send.ts

```diff
@@ -130,65 +135,38 @@ export class Send extends IronfishCommand {
       this.exit(1)
     }
 
-    if (fee == null) {
-      let suggestedFee = ''
-      try {
-        const response = await client.estimateFee({
-          fromAccountName: from,
-          receives: [
-            {
-              publicAddress: to,
-              amount: CurrencyUtils.encode(amount),
-              memo: memo,
-            },
-          ],
-        })
-
-        switch (flags.priority) {
-          case 'low':
-            suggestedFee = CurrencyUtils.renderIron(response.content.low)
-            break
-          case 'high':
-            suggestedFee = CurrencyUtils.renderIron(response.content.high)
-            break
-          default:
-            suggestedFee = CurrencyUtils.renderIron(response.content.medium)
-        }
-      } catch {
-        suggestedFee = ''
-      }
-
-      const input = await CliUx.ux.prompt(
-        `Enter the fee amount in $IRON (min: ${CurrencyUtils.renderIron(
-          1n,
-        )} recommended: ${suggestedFee})`,
-        {
-          required: true,
-          default: suggestedFee,
-        },
-      )
-
-      fee = CurrencyUtils.decodeIron(input)
-    }
-
-    if (fee < 1n) {
-      this.error(`The minimum fee is ${CurrencyUtils.renderOre(1n, true)}`)
-    }
-
     if (expiration !== undefined && expiration < 0) {
       this.log('Expiration sequence must be non-negative')
       this.exit(1)
     }
 
+    const createResponse = await client.createTransaction({
+      fromAccountName: from,
+      receives: [
+        {
+          publicAddress: to,
+          amount: CurrencyUtils.encode(amount),
+          memo,
+          assetId,
+        },
+      ],
+      fee: flags.fee,
+      feePriorityLevel: flags.priority,
+      expiration: expiration,
+    })
+
+    const rawTransactionBytes = Buffer.from(createResponse.content.transaction, 'hex')
+    const rawTransaction = RawTransactionSerde.deserialize(rawTransactionBytes)
+
     if (!flags.confirm) {
       this.log(`
 You are about to send:
 ${CurrencyUtils.renderIron(
-  amount,
+  rawTransaction.spends[0].note.value(),
```

Comment:
How should I get amount from Raw Transaction? sum up all the notes values?

Reply:
We can use the amount that the user inputs

To get the amount from the raw transaction I think we'd need to iterate over the notes in `rawTransaction.receives` and exclude any that the sender is receiving as change. It might not be possible to distinguish those notes though: if an account sends a transaction to itself then all of the notes will belong to that account.

Reply:
I see, okay!